components/ProgressView/README.md

```diff
@@ -187,3 +190,20 @@ tracking its development:
 
 - Task: [Implement a color themer](https://www.pivotaltracker.com/story/show/157095443)
 
+
+## Accessibility
+
+<!-- Extracted from docs/accessibility.md -->
+
+### MDCProgressView
+
+#### -accessibilityValue
+
+Like UIProgressView, MDCProgressView's `accessibilityValue` is based on the current value of the ProgressView's
+`progress` property. UIProgressView has internal logic that makes it so that a `progress` value of .476 will yield an
```

Comment:
Nit: I think we can drop the example, and just document that `accessibilityValue` is expected to match that of `UIProgressView` for the same configurations. What do you think? 

Reading the example makes me wonder why ".476" is "fifty percent" and not "forty-seven percent" and might be more distracting than it is informative.